src/test/java/cs2103/v15_1j/jimjim/JJStorageTest.java

```diff
@@ -0,0 +1,28 @@
+package cs2103.v15_1j.jimjim;
+
+import static org.junit.Assert.*;
+
+import java.io.FileNotFoundException;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class JJStorageTest {
+	private JJStorage storage;
+
+	@Before
+	public void setUp() throws Exception {
```

Comment:
I don't think a test should throw an exception, since it is supposed to test the behavior. The behavior is either correct or wrong. So if the test throws an expected exception, it is correct behavior. Otherwise it is wrong?